### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by jponge**

### CODE REVIEW

**Comment by jponge** [comment]
Work in progress

File: _posts/2015-01-20-golo-2.0.0.markdown [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+![Auvergne FTW](https://farm9.staticflickr.com/8627/16138075430_03a6458597_c.jpg)
+
+Golo 2.0.0 is here with new language features, improvements and fixes. This is the result of a great
+community effort: we are glad that new contributors joined us, and the new language features are
+contributions developped outside of the [Dynamid](http://dynamid.citi-lab.fr/) research group.
+
+As Golo tries to stick to the semantics of [semantic versioning](http://semver.org), we bumped to a
+new major version. Indeed, *named augmentations* are a new language feature, and they turn `with`
+into a keyword, which introduces a tiny forward incompatibility.
```
Comment (yloiseau) [inline _posts/2015-01-20-golo-2.0.0.markdown]: isn't *backward* incompatibility?

File: _posts/2015-01-20-golo-2.0.0.markdown [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+{% endhighlight %}
+
+Then, such augmentations can be applied to types as in:
+
+{% highlight golo %}
+augment java.util.Collection with FooBar
+augment MyStruct with Spamable
+augment java.lang.String with FooBar, Spamable
+{% endhighlight %}
+
```
Comment (yloiseau) [inline _posts/2015-01-20-golo-2.0.0.markdown]: maybe we could add a direct link to the corresponding documentation section?

File: _posts/2015-01-20-golo-2.0.0.markdown [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+which expands to:
+
+{% highlight golo %}
+function decorator =  |func| -> |x| -> func(x)
+
+function identity = |x| -> decorator!(|x| -> x)(x)
+{% endhighlight %}
+
```
Comment (yloiseau) [inline _posts/2015-01-20-golo-2.0.0.markdown]: idem about direct link

File: _posts/2015-01-20-golo-2.0.0.markdown (hunk 1 above)
Comment (jponge) [inline _posts/2015-01-20-golo-2.0.0.markdown]: :+1:

File: _posts/2015-01-20-golo-2.0.0.markdown (hunk 2 above)
Comment (jponge) [inline _posts/2015-01-20-golo-2.0.0.markdown]: :+1:

**Comment by Mogztter** [comment]
\o/

**Comment by jponge** [comment]
@Mogztter :smiley:

**Pull request merged**